__init__.py

```diff
@@ -40,14 +40,17 @@
 from config import get_config, initialize
 initialize(bl_info["version"], __file__)
 
+from login_token_cache import initialize
+initialize(__file__)

```

Comment:
Which `initialize` are you calling here?  There's another `initialize` function imported a couple lines above this.

Reply:
The functions require a different number of parameters so I assume python is aware the signature is different and is calling the correct one. This is confusing though so I'll do it a different way.